### ISSUE TITLE
UNREGISTERED error

### ISSUE THREAD

**Issue by kushpr** [jonathandata1/atsend#3]
+cme error:pacm (AP), UNREGISTERED

can i bypass this ?

**Comment by jonathandata1** [comment]
what happens when you go to your dialer and you press

*#0808#

Do you see DM,ADB?

I can see your VID:PID is 04E8:6862

**Comment by mtklover** [comment]
Could you incorporate android_identification_at_commands_samsung_linux.sh
into atsend ?
Its a really good needy tool
@jonathandata1

**Comment by mtklover** [comment]
Usally get this response when command doesnt exist (PACM(AP),UNREGISTED)

I think Tyr supports AT command dumping from system right @jonathandata1 ?
Samsung uses theyre own kind of AT Commands

**Comment by jonathandata1** [comment]
> Usally get this response when command doesnt exist (PACM(AP),UNREGISTED)
> 
> I think Tyr supports AT command dumping from system right @jonathandata1 ? Samsung uses theyre own kind of AT Commands

Yes it does, you can use https://github.com/jonathandata1/tyr to dump the AT commands

**Comment by jonathandata1** [comment]
> 

atsend has a built in general command tool, when you open ATSEND type help
